paddle/majel/README.md

```diff
@@ -95,32 +95,170 @@ Because `variant` may be thought of as "multi-type, single value", we can utiliz
 
 ## implement Tensor in Paddle
 
+We aim to implement an independent tensor library, which provides flexiable and efficient tensor operations and can run on muti-devices(CPU and GPU). In brief, there are mainly two aspects to consider. First, we have to make encapsulation of the hardware for the tensor library, including device memory, computing handles, cuda streams and so on. Second, after the tensor libaray is able to access the hardware resources, we should make carefully design and abstraction of the upper levels and provide flexiable interfaces.  
```

Comment:
The paragraph is not dense enough.  For example, "In brief, there are mainly two aspects to consider. " doesn't seem necessary.  An example change is as follows:

> We want to create a Tensor class to replace Vector and Matrix, and to support high-dimensional data. We want to accelerate operations on Tensor using CUDA, so we need CPU and GPU memory allocation and CUDA kernel launching. We also want to make sure that the Tensor interface is friendly to its callers.